certinject/cryptoapi_windows.go

```diff
@@ -0,0 +1,181 @@
+package certinject
+
+import "golang.org/x/sys/windows/registry"
+import "crypto/sha1"
+import "encoding/hex"
+import "strings"
+import "math"
+import "time"
+
+// In 64-bit Windows, this key is shared between 64-bit and 32-bit applications.
+// See https://msdn.microsoft.com/en-us/library/windows/desktop/aa384253.aspx
+const cryptoApiCertStoreRegistryBase = registry.LOCAL_MACHINE
+const cryptoApiCertStoreRegistryKey = `SOFTWARE\Microsoft\EnterpriseCertificates\Root\Certificates`
+const cryptoApiMagicName = "Namecoin"
+const cryptoApiMagicValue = 1
+
+func injectCertCryptoApi(derBytes []byte) {
+
+	// Format documentation of Microsoft's "Certificate Registry Blob":
+	
+	// 5c 00 00 00 // propid
+	// 01 00 00 00 // unknown
```

Comment:
Quite possibly.  None of the reverse-engineered documentation that I found explained what the field was for or what format it used.  In any event its value seems to be constant everywhere, so reverse-engineering what it's for doesn't seem necessary.